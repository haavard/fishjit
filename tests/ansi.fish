cf+"[1m"cf+"[34m><>"arv
o      ;!?     l      <
