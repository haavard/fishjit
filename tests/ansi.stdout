[1m[34m><>
